### PR TITLE
Add/660-async-locator-click

### PR DESCRIPTION
**Pull request by inancgumus**
This PR is necessary to have feature parity with `page` and `elementHandle` `click` as they are async. However, `locator.click` is not. This confuses users.

Related: grafana/xk6-browser#311 and grafana/xk6-browser#439
Updates: grafana/k6#4490
Closes: grafana/xk6-browser#660

### CODE REVIEW

File: common/locator.go [hunk 1]
```diff
@@ -31,21 +31,17 @@ func NewLocator(ctx context.Context, selector string, f *Frame, l *log.Logger) *
 }
 
 // Click on an element using locator's selector with strict mode on.
-func (l *Locator) Click(opts goja.Value) {
+func (l *Locator) Click(opts goja.Value) error {
 	l.log.Debugf("Locator:Click", "fid:%s furl:%q sel:%q opts:%+v", l.frame.ID(), l.frame.URL(), l.selector, opts)
 
-	var err error
-	defer func() { panicOrSlowMo(l.ctx, err) }()
```
Comment (ankur22) [inline common/locator.go:38]: Do we not need `slowMo` for `click`?

**Review by ankur22 — COMMENTED** [review]
LGTM, apart from the missing `slowMo` option, is that not needed? I remember we had an issue that detailed APIs where `slowMo` was to be removed, but i can't seem to find it.

File: common/locator.go (hunk 1 above)
Comment (inancgumus) [reply common/locator.go:38]: Nice catch, let me add it back.

Comment (inancgumus) [reply common/locator.go:38]: Resolved in https://github.com/grafana/xk6-browser/blob/ab096eb7a09e13646df318a69ed8abb0e6514761/common/locator.go#L44-L46.

**Review by ankur22 — APPROVED** [review]
LGTM

**Review by ka3de — APPROVED** [review]
LGTM.

**Pull request merged**